### PR TITLE
RD-3974: Added delay to the response of the request

### PR DESCRIPTION
**Pull request by NorbertAmartus**
## Description

Changes consists of delaying the response for one of the requests, so that we could be certain that cypress will have a necessary time for detecting the `<Loader />` component on the screen.

## Screenshots / Videos

N/A

## Checklist

- [x] My code follows the UI Code Style.
- [x] I verified that all tests and checks have passed.
- [ ] I verified if that change requires a documentation update.
- [x] I added proper labels to this PR.

## Tests

 

I've run the cypress test file, related to the jira issue, multiple times to see if the issue is gone for good 🚀 
And it seems like 250ms delay for the response is creating a safe space for the cypress to confirm that the `<Loader />` is visible on the screen!

[System tests](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/1792) has been executed 
(All tests are passing, except 2 which are failing due to the problems detected on the master branch)

## Documentation

N/A

### CODE REVIEW

File: test/cypress/integration/user_menu_spec.ts [hunk 1]
```diff
@@ -18,6 +18,14 @@ describe('User Menu', () => {
         cy.contains(optionName).should('not.exist');
     };
 
+    const delayTenantsRefresh = (delayTime: number) => {
+        cy.intercept('/console/sp/tenants?_include=name&_get_all_results=true', request => {
```
Comment (qooban) [inline test/cypress/integration/user_menu_spec.ts:22]: Optional:

I'm wondering if you need to be so specific about the actual request (check all query parameters), but not to 
If necessary maybe worth to consider using a bit more programmer-friendly construction:

cy.intercept({
  pathname: '/console/sp/tenants',
  query: { _include: 'name', _get_all_results: 'true' } /* optional */
}, request => {

I don't want to unnecessarily slow down merging this fix, so it's an optional comment.

**Review by qooban — APPROVED** [review]
Looks good! 👍🏻 

Added just one optional comment.

File: test/cypress/integration/user_menu_spec.ts (hunk 1 above)
Comment (NorbertAmartus) [reply test/cypress/integration/user_menu_spec.ts]: Sure, that looks like a neat fix 😉 

And thanks for taking into the account the available velocity related to merging thix fix 😎

Comment (NorbertAmartus) [reply test/cypress/integration/user_menu_spec.ts]: Changes has been added 🎉

Comment (NorbertAmartus) [reply test/cypress/integration/user_menu_spec.ts]: (Just for the sake of the information, I've tested this solution locally and it seems to be working as expected -> perfectly fine 😎 )

**Review by qooban — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**